alacritty.yml

```diff
@@ -481,10 +482,13 @@
 #
 #   - ToggleViMode
 #   - SearchForward
+#       Start searching to the right of the search origin.
 #   - SearchBackward
+#       Start searching to the left of the search origin.
```

Comment:
I see what you are saying. You saying that just because an object is the "to the left" of another, it does not mean the object is "heading in the left direction".

I'll replace "to the left" with "towards the left". The words `leftward` and `rightward` would also work here, but `towards the left` is simpler English, so I'll stick with that.